dask_planner/src/sql.rs

```diff
@@ -108,6 +108,11 @@ impl ContextProvider for DaskSQLContext {
             let rtf: ReturnTypeFunction = Arc::new(|_| Ok(Arc::new(DataType::Int64)));
             return Some(Arc::new(ScalarUDF::new("year", &sig, &rtf, &fun)));
         }
+        if "cot".eq(name) {
```

Comment:
Question mainly for @jdye64. Do you have a preference on adding some functions to the dialect in rust vs registering it as a udf on the python side by creating a DaskFunction?

Reply:
I would prefer to have certain functions, like `cot`, which are quite common and in most DB engines be in the Rust dialect. However, once the Dialect is completed you will be able to interact with it through Python and add new `udfs` of course.